reactjs-films-homework/src/components/MovieCard/MovieCard.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { MovieCard } from './MovieCard';
+import { IntlProvider } from 'react-intl';
+import { BrowserRouter } from 'react-router-dom';
+import messages from '../../shared/localization/messages';
+
+describe('MovieCard component', () => {
+  it('MovieCard component should render correctly', () => {
```

Comment:
make const defaultProps with default data and pass into component like this {...defaultProps}. You can use this in some it. just put this const directly after describe